tests/data/config/_init.yaml

```diff
@@ -1,4 +1,5 @@
 # Default configuration for tests.
+s3instance: none
```

Comment:
If you agree with the change I proposed in the config schema, this could be 

```
s3instance: null
```

Reply:
Implemented.